examples/rust/file_transfer/tests/100_32_test.bin

```diff
@@ -0,0 +1,2 @@
+�u���P��/f�
```

Comment:
@etorreborre 😃

Reply:
Fixed. Not a big fan of tests writing to the source directory. I'll come back to that at some stage